sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/it/SpannerChangeStreamIT.java

```diff
@@ -151,7 +152,8 @@ public void testReadSpannerChangeStreamImpl(TestPipeline testPipeline, String ro
                     .withMetadataDatabase(ENV.getMetadataDatabaseId())
                     .withMetadataTable(metadataTableName)
                     .withInclusiveStartAt(startAt)
-                    .withInclusiveEndAt(endAt))
+                    .withInclusiveEndAt(endAt)
+                    .withWatermarkRefreshRate(Duration.standardSeconds(1)))
```

Comment:
Seems this could be removed and just use the default once it is set.

Reply:
Done